Base/res/welcome.txt

```diff
@@ -26,11 +26,11 @@ up to your monitor.
 $ /res/icons/16x16/filetype-html.png
 > Connect to the Internet!
 On the Internet, you can correspond through electronic mail (e-mail), get the
-latest news and financial information, and visit Web sites around the world,
+latest news and financial information, and visit websites around the world,
```

Comment:
Just... no. This text imitates (and parodies) 90s OS introduction texts. It's "the Internet", "Web sites" and so on.